phpstan-baseline.neon

```diff
@@ -1,2 +1,7 @@
 parameters:
-    ignoreErrors:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#1 \\$input of class AsyncAws\\\\Ses\\\\ValueObject\\\\EmailContent constructor expects array\\(\\)\\|array\\('Simple' \\=\\> array\\|AsyncAws\\\\Ses\\\\ValueObject\\\\Message\\|null, 'Raw' \\=\\> array\\|AsyncAws\\\\Ses\\\\ValueObject\\\\RawMessage\\|null, 'Template' \\=\\> array\\|AsyncAws\\\\Ses\\\\ValueObject\\\\Template\\|null\\), array\\('Raw' \\=\\> AsyncAws\\\\Ses\\\\ValueObject\\\\RawMessage\\) given\\.$#"
+			count: 1
+			path: src/Integration/Laravel/Mail/src/Transport/AsyncAwsSesTransport.php
```

Comment:
I cannot reproduce this error here: https://phpstan.org/r/e109b47f-d7e5-482f-920d-e659b0d945e7